commons-db/src/main/java/com/palantir/nexus/db/sql/SqlConnectionHelper.java

```diff
@@ -212,15 +214,21 @@ Long selectLongWithDefault(Connection c, String key, Long defaultVal, Object...
 
     AgnosticLightResultSet selectLightResultSetUnregisteredQuery(Connection c,
                                                                  String sql,
-                                                                 Object... vs)
-            throws PalantirSqlException, PalantirInterruptedException {
```

Comment:
Because these can be thrown by pretty much every method in DbKvs, so there is no point really in declaring them. It just makes the code harder to maintain because you don't really know if the method still throws those exceptions, or the "throws" clause is now totally out of date because someone changed the impl. There is no enforcement from the compiler for unchecked exceptions